source/404.md

```diff
@@ -5,3 +5,12 @@ containerClass: not-found
 ---
 
 # 404 Page not Found
+
+Well darn&hellip; we can't find the page you're looking for. One of these links might help.
+
+* {% fa fa-angle-right %} {% url 'Introduction to Cypress' introduction-to-cypress %}
+* {% fa fa-angle-right %} {% url 'The API docs' api %}
+* {% fa fa-angle-right %} {% url 'Tutorials' tutorials %}
+* {% fa fa-angle-right %} {% url 'The Cypress Dashboard' https://www.cypress.io/dashboard/ %}
+
+> Also, {% url Gleb https://glebbahmutov.com/ %} is a legend. Just sayin'.
```

Comment:
_:: crosses fingers... if I believe it might just survive this PR to go live ::_ :smile:

Cypress is a team effort. I tried fitting the names of Cypress titans like Jennifer, Chris, Magnolia, and Brian on there -- and everyone else -- but it looked a bit busy.

![hi5](https://user-images.githubusercontent.com/1855109/44999089-ac9fe680-afb2-11e8-9691-17c1615a2007.png)